tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -834,7 +834,7 @@ func computeIterations(run *v1alpha1.Run, tls *pipelineloopv1alpha1.PipelineLoop
 			}
 		}
 		if p.Name == "step" {
-			step, _ = strconv.Atoi(p.Value.StringVal)
+			step, err = getIntegerParamValue(p)
```

Comment:
Should we throw an error if the `step` is not defined correctly?

Reply:
in fact, if an error happened, the step will be 0, and we throw the error in the last. 
```
if step == 0 {
			return 0, iterationElements, fmt.Errorf("invalid values step: %d found in runs", step)
		}
```

Anyway, I addressed your review comments to make the code clear.